addons/a11y/src/components/A11YPanel.tsx

```diff
@@ -48,7 +48,8 @@ const Incomplete = styled.span(({ theme }) => ({
 
 const Loader = styled(({ className }) => (
   <div className={className}>
-    <Icon inline icon="sync" status="running" /> Please wait while a11y scan is running ...
+    <Icon inline icon="sync" status="running" /> Please wait while the accessibility scan is running
```

Comment:
a11y or accessibility?

I'm leaning towards ally

Reply:
Hey @wuweiweiwu thanks for reviewing! Previously, we had been using “a11y” as the abbreviation for accessibility but we are trying to get away from using that on the display. The reason being that we, as developers, may understand what a11y means but many who are new to the space or haven't seen an abbreviation like that may not. Since this platform could be used by people who are not developers and to keep consistency from the visible text perspective, I suggest that we keep it as “accessibility”.